oneflux/local_settings_template.py

```diff
@@ -0,0 +1,31 @@
+'''
+oneflux.configs_local
+
+For license information:
+see LICENSE file or headers in oneflux.__init__.py 
+
+Pipeline optional configuration parameters
+
+@author: Gilberto Z. Pastorello
+@contact: gzpastorello@lbl.gov
+@date: 2021-09-01
+'''
+import sys
+
+# # Configs for ONEFlux running for: FLUXNET sites, creating FLUXNET2015 data product, using ERA-Interim downscaling
+# MODE_ISSUER = 'FLX'
+# MODE_PRODUCT = 'FLUXNET2015'
+# MODE_ERA = 'ERAI'
+# ERA_FIRST_YEAR = 1989
+# ERA_LAST_YEAR = 2014
+
+# Configs for ONEFlux running for: FLUXNET sites, creating FLUXNET data product, using ERA-5 downscaling
+MODE_ISSUER = 'FLX'
```

Comment:
This is good, however it would be important to define.
1) if must be only 3 chartacters or can be 4
2) to add as comment the possible options so that they are interpretable... for example AMF, EUF, OZF. For ICOS if it can not be 4 chars then could be ICS or ETC (to decide). Then also describe when we should use FLX because it would become basically unused.

Reply:
Thanks @darpap, these reviews will more commonly involve people doing the coding.

The most recent commit ([7ce1b05](https://github.com/fluxnet/ONEFlux/pull/20/commits/7ce1b0576f4a71ee4df029a0c3fcb8b652260f0e)) now includes instructions to use local_settings.py file, examples of values for more networks/data products, and an updated last year for ERA for 2021.

We currently have file formatting rules expecting a 3-character issuer code. We can consider changing this in the future, but will keep this format for this round.